ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,61 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as Ready in the Design System.
+
+- [ ] Documentation is complete. [Documentation writing info](#)
+    - Read through documentation and make sure the following is done
+      - [ ] Description of component or pattern
+      - [ ] HTML code snippet example(s)
+      - [ ] React code snippet example(s)
+      - [ ] All react properties are explained
+      - [ ] Guidance on when to use this item
+      - [ ] Guidance on when not to use this item
+      - [ ] Accessibility considerations
+
+---
+
+- [ ] Validate HTML
+  - Go to [validator.w3.org](https://validator.w3.org/#validate_by_input) paste your code in and make sure it validates without error.
+
+---
```

Comment:
I'd like to drop another section in here for Automated Accessibility Scanning, with bullet points referencing the direct links to Chrome and Firefox plugins on Deque's website ( https://www.deque.com/axe ). The component should not throw any Violations, and ideally, no Best Practices warnings. Some of the Best Practices can be ignored like `Means to skip repeated blocks` of content, or `No main section` because they generally don't apply to non-page-level components.

The e2e tests I am putting in with PR https://github.com/CMSgov/design-system/pull/308 avoid `Best Practices` for this very reason. I might go back and tweak that later to avoid certain rules, but for now, we want components to pass Section 508, WCAG2 A, and WCAG2 AA. Bonus points for any WCAG2.1 A | AA passes, which are baked into the latest versions of aXe as well.

Reply:
Do we want to encourage both pasting codes into the validator as well as using one of the plugins or one of the 2?

Reply:
☝️@1Copenut

Reply:
👋